pbench_copy_data

```diff
@@ -85,5 +85,5 @@ if [[ "$results_prefix" == *"--"* ]]; then
 	exit 1
 fi
 
-pbench-copy-results --controller="${controller}" --user="${user}" --prefix="${results_prefix}"
+pbench-copy-results --controller="${controller}" --user="${user}" --controller="${results_prefix}"
```

Comment:
The variable name, `results_prefix`, is a bit misleading.  Could that be renamed to `controller` to avoid confusion and increase maintainability?

Reply:
Changed.

Reply:
Actually why do we want to specify controller twice.  We use prefix to provide more information, --controller for the top level.

Reply:
I think the suggestion was sed s/results_prefix/controller/.  In this case we're saying the controller _is_ results_prefix - we're effectively changing the meaning of "controller" to better organize the results.  results_prefix is used in other places throughout zathras though, so I don't see a gain (other than immediate readability) to have a controller var whose contents are identical to results prefix here.  Either that or @portante wants something else entirely...